### PR TITLE
Help Center: Update copy

### PR DESCRIPTION
**Pull request by escapemanuele**
## Proposed Changes

- Update the copy.
- String freeze to have translations ongoing

## Testing Instructions

*

### CODE REVIEW

File: packages/odie-client/src/components/message/dislike-feedback-message.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const DislikeFeedbackMessage = () => {
 						} }
 					>
 						{ __(
-							'I’m sorry my last response didn’t meet your expectations! Here’s some other ways to get more in-depth help:',
+							'Let’s get the information you need. Would you like to contact our support team?',
```
Comment (agrullon95) [inline packages/odie-client/src/components/message/dislike-feedback-message.tsx]: @escapemanuele , @lucasmendes-design 
`support` is not capitalized here, but below in `ODIE_ERROR_MESSAGE`, `support` is capitalized. Is this intentional?

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16988170

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @escapemanuele, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Great! Thanks.`
- `Not what you were looking for?`
- `We appreciate your feedback.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16988170).

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16988170).

**Pull request merged**